release notes/v0.48.0.md

```diff
@@ -8,6 +8,9 @@ k6 v0.48.0 is here 🎉! This release includes:
 - [#3350](https://github.com/grafana/k6/pull/3350) In k6 [v0.37](https://github.com/grafana/k6/releases/tag/v0.37.0), we deprecated a gRPC Invoke's parameter `headers`. This release removes the parameter. If you need to set headers, use a `metadata` parameter instead.
 - [#3448](https://github.com/grafana/k6/pull/3448) metric names now need to conform to both otel and prometheus name requirements while still being limitted to 128 characters (old k6 limit).
 - [#3451](https://github.com/grafana/k6/pull/3451) In k6 [v0.25](https://github.com/grafana/k6/releases/tag/v0.25.0) module specifiers were allowed to be urls with scheme, and k6 prepending `https://` to anything that it wasn't recognizing was deprecated. From this version k6 will no longer try to do that all, please use full URLs if you want to load remote modules.
+- [#3389](https://github.com/grafana/k6/pull/3389) In k6 v0.38 we deprecated `--logformat` introducing the better alternative `--log-format`. This realease removes `--logformat` flag.
+- [#3390](https://github.com/grafana/k6/pull/3390) In k6 [v0.35](https://github.com/grafana/k6/releases/tag/v0.35.0) we deprecated all CSV output's CLI arguments having a not consistent case with other outputs. This realease removes definitely them.
+- [#3365](https://github.com/grafana/k6/pull/3365) In k6 [v0.41](https://github.com/grafana/k6/releases/tag/v0.41.0) we deprecated the `k6 convert` CLI command. This realease deleted it, the better [har-to-k6](https://github.com/grafana/har-to-k6) alternative should be used.
```

Comment:
A minor grammar and keeping the link to the release notes, just for the consistency

```suggestion
- [#3389](https://github.com/grafana/k6/pull/3389) In k6 [v0.38](https://github.com/grafana/k6/releases/tag/v0.38.0) we deprecated `--logformat` introducing the better alternative `--log-format`. This release removes `--logformat` flag.
- [#3390](https://github.com/grafana/k6/pull/3390) In k6 [v0.35](https://github.com/grafana/k6/releases/tag/v0.35.0) we deprecated all CSV output's CLI arguments having a not consistent case with other outputs. This release removes them.
- [#3365](https://github.com/grafana/k6/pull/3365) In k6 [v0.41](https://github.com/grafana/k6/releases/tag/v0.41.0) we deprecated the `k6 convert` CLI command. This release deleted it, the better [har-to-k6](https://github.com/grafana/har-to-k6) alternative should be used.
```